NEWS.md

```diff
@@ -25,6 +26,60 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [0.7.0] - 2023-12-22
+
+The main effort in this release has gone into porting it to wlroots 0.17
+and tidying up regressions.
+
+# Added
+
+- Support titlebar hover icons. Written-by: @spl237
+- Add theme options osd.workspace-switcher.boxes.{width,height}
+  Written-by: @kyak
+- Add actions `VirtualOutputAdd` and `VirtualOutputRemove` to control virtual
+  outputs. Written-by: @kyak (#1287)
+- Teach MoveToEdge to move windows to adjacent outputs.
+  Written-by: @ahesford
+- Implement `<font place="InactiveWindow">`. Written-by: @ludg1e (#1292)
+- Implement cursor-shape-v1 protocol to allow Wayland clients to request a
+  buffer for a cursor shape from a compositor. Written-by: @heroin-moose
+- Implement fractional-scale-v1 protocol to allow Wayland clients to properly
+  scale on outputs with fractional scale factor. Written-by: @heroin-moose
+- Add ResizeTo action (#1261)
+- Allow going backwards in window-switcher OSD by using arrow-up or arrow-left.
+  Written-by: @jg6766
```

Comment:
Thanks, though I would prefer @jp7677 here :)

Reply:
Apologies. I'm not sure what made we write jg6766... Now fixed.